ReactAndroid/src/main/java/com/facebook/react/views/toolbar/ReactToolbarManager.java

```diff
@@ -84,6 +85,13 @@ public void setTitle(ReactToolbar view, @Nullable String title) {
     view.setTitle(title);
   }
 
+  @ReactProp(name = "rtl")
+  public void setLayoutDirection(ReactToolbar view, Boolean flag) {
+    if (flag.equals(true)) {
```

Comment:
in case of null It simply won't do anything and the layout can be controlled by parent layout. Currenly I'm using this feature in my app